nsbno/spor
# Fix typo "icons" -> "stars"

Partial PR: the important changes are done but I'm not well enough versed in your process to finalise it for merging so feel free to take over or give me pointers.

This is a breaking change without backwards compatibility and will require a semver major version bump.

## Background

The icons for stars in Spor are named "icons", which is probably a typo and makes them hard to find. This fixes the typo in code – they should probably also be renamed in Figma. I made a comment there too.

## Solution

Renamed the star svgs to be called Stars instead of Icons. Elm, React and React Native icons will need to be regenerated if that process isn't automatic.

## UU checks


Note: To trigger pipeline for the documentation site (spor.vy.no) you need to bump the version.

## How to test

Build icons and check that the star icons are now called "Stars" instead of "Icons"

## Screenshots

| Before | After |
| ------ | ----- |
| ![image](https://github.com/user-attachments/assets/7251edf2-ac6f-4ed0-a576-307568627960) |       |
